docs/operator-manual/high_availability.md

```diff
@@ -61,7 +61,7 @@ number of allowed concurrent kubectl fork/execs.
 * The controller uses Kubernetes watch APIs to maintain lightweight Kubernetes cluster cache. This allows to avoid querying Kubernetes during app reconciliation and significantly improve
 performance. For performance reasons controller monitors and caches only preferred the version of a resource. During reconciliation, the controller might have to convert cached resource from
 preferred version into a version of the resource stored in Git. If `kubectl convert` fails because conversion is not supported then controller falls back to Kubernetes API query which slows down
-reconciliation. In this case advice user-preferred resource version in Git.
+reconciliation. In this case advice to use preferred resource version in Git.
```

Comment:
```suggestion
reconciliation. In this case, we advise you to use the preferred resource version in Git.
```

Would this still make sense? The grammar is a bit clearer to me.

Reply:
Thanks @crenshaw-dev
Updated.